setup.py

```diff
@@ -32,7 +32,7 @@ def _get_long_description():
     author_email="opensource@faculty.ai",
     url="https://github.com/facultyai/dash-bootstrap-components",
     packages=find_packages(),
-    install_requires=["dash>=0.32.1", "dash-html-components"],
+    install_requires=["dash>=0.40.0"],
```

Comment:
`dash` now pins the versions of its various dependencies, including `dash-html-components`, so there is no need for us to specify it as a separate requirement.